src/expression/definitions/interpolate.ts

```diff
@@ -126,14 +126,15 @@ class Interpolate implements Expression {
             if (stops.length && stops[stops.length - 1][0] >= label) {
                 return context.error('Input/output pairs for "interpolate" expressions must be arranged with input values in strictly ascending order.', labelKey) as null;
             }
-
             const parsed = context.parse(value, valueKey, outputType);
             if (!parsed) return null;
             outputType = outputType || parsed.type;
             stops.push([label, parsed]);
         }
 
         if (!verifyType(outputType, NumberType) &&
+            !verifyType(outputType, StringType) &&
```

Comment:
This looks wrong code-wise - we should not support arbitrary string interpolation.

Reply:
We do support arbitrary values for `step` , and I'm not sure of the discrepancy between the two.

Reply:
Let's not introduce string interpolation as part of this PR. If there's a need for it, let's add it in the future.

Reply:
OK, i'll attempt to add back the .projection() interpolation